datafusion/physical-plan/src/unnest.rs

```diff
@@ -299,7 +296,7 @@ impl UnnestStream {
                         continue;
                     };
                     self.metrics.output_batches.add(1);
```

Comment:
I verified that BaselineMetrics only records number of rows, not number of batches